From 4daf82c991323e769d46fa2d8fa1252addb0627f Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Wed, 3 Oct 2018 20:22:55 -0700
Subject: [PATCH 6/8] DO-NOT-UPSTREAM: soc: rockchip: Disable
 -Wshift-count-negative for pm_domains.o

GitHub issue: https://github.com/ClangBuiltLinux/linux/issues/92
LLVM bug: https://bugs.llvm.org/show_bug.cgi?id=38789
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/soc/rockchip/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/soc/rockchip/Makefile b/drivers/soc/rockchip/Makefile
index afca0a4c4b72..79cfdbd6336b 100644
--- a/drivers/soc/rockchip/Makefile
+++ b/drivers/soc/rockchip/Makefile
@@ -4,3 +4,5 @@
 #
 obj-$(CONFIG_ROCKCHIP_GRF) += grf.o
 obj-$(CONFIG_ROCKCHIP_PM_DOMAINS) += pm_domains.o
+
+CFLAGS_pm_domains.o += $(call cc-disable-warning, shift-count-negative)
-- 
2.22.0.rc1

